src/libraries/Microsoft.PowerFx.Core/Utils/TextFormatUtils.cs

```diff
@@ -25,7 +26,9 @@ public static bool IsValidFormatArg(string formatArg, out bool hasDateTimeFmt, o
                 }
             }
 
-            hasDateTimeFmt = formatArg.IndexOfAny(new char[] { 'm', 'M', 'd', 'D', 'y', 'Y', 'h', 'H', 's', 'S', 'a', 'A', 'p', 'P' }) >= 0;
+            hasDateTimeFmt = (formatArg.IndexOfAny(new char[] { 'm', 'M', 'd', 'D', 'y', 'Y', 'h', 'H', 's', 'S', 'a', 'A', 'p', 'P' }) >= 0) 
+                && (formatArg.ToLower(CultureInfo.InvariantCulture).IndexOf("us", StringComparison.Ordinal) < 0);
```

Comment:
Can this be ignore case? and then we can avoid ToLower call?

Reply:
yes, update